bin/imports.js

```diff
@@ -14,6 +14,7 @@ const findUp = require('find-up');
 const path = require('path');
 const fs = require('fs');
 const Module = require('module');
+const substrings = ['alpha', 'beta', 'rc'];
```

Comment:
you can make this more generic
```suggestion
const substrings = ['-', '+'];
```
the major.minor.patch will always be separated from everything else by a dash or a plus https://semver.org/#backusnaur-form-grammar-for-valid-semver-versions